cppfd/boundaryconditions.cpp

```diff
@@ -1,17 +1,5 @@
 #include"boundaryconditions.h"
 
-BoundaryConditions::BoundaryConditions(Mesh& m, std::map<std::string, double> velocity_values)
-  : mesh(m)
-  , u_top(velocity_values["u_top"])
```

Comment:
I would rename these member variables (`u_<location>` and `v_<location>`) to make them less confusing as `v` might be confused with the velocity itself, while it only denotes its second spatial component. 
Suggestion: `velocity_x_top` for `u_top` and `velocity_y_top` for `v_top` (and similarly for other locations)>

Reply:
Making this a separate issue for later.